radix-engine-common/src/data/manifest/mod.rs

```diff
@@ -40,3 +40,9 @@ pub fn manifest_decode<T: ManifestDecode>(buf: &[u8]) -> Result<T, DecodeError>
     ManifestDecoder::new(buf, MANIFEST_SBOR_V1_MAX_DEPTH)
         .decode_payload(MANIFEST_SBOR_V1_PAYLOAD_PREFIX)
 }
+
+pub fn manifest_transcode<T: ManifestEncode + ?Sized, P: ManifestDecode>(
```

Comment:
Really not  sure if this is correct approach.
Did not come to any better idea how to convert `Vec<u8>` to `ManifestValue`.

Reply:
This is the easiest. Only only implemented `bytes <> rust value` and `bytes <> SBOR value` among bytes, rust value and SBOR value, see [here](https://radixdlt.atlassian.net/wiki/spaces/S/pages/2905374962/REP-47+SBOR+Upgrade+For+Better#Why-SBOR-in-the-first-place%3F).

Can you rename it to `to_manifest_value` and add an additional one `from_manifest_value() -> Result<T, DecodeError>`?

Was also thinking if we should add similar constructs for Scrypto  SBOR.

Reply:
It looks like both functions `to_manifest_value()` and `from_manifest_value()` are the same in fact (bit surprised here, unless I am missing smth) so maybe I will create only one `to_and_from_manifest_value()` ?